packages/grafana-data/src/field/fieldOverrides.test.ts

```diff
@@ -46,8 +46,16 @@ const property3 = {
   shouldApply: () => true,
 } as any;
 
+const shouldApplyFalse = {
+  id: 'custom.shouldApplyFalse', // Match field properties
+  path: 'shouldApplyFalse', // Match field properties
+  isCustom: true,
+  process: (value: any) => value,
+  shouldApply: () => false,
+} as any;
```

Comment:
nit: to avoid the strict null error but would be better if we use the actual type instead of `any`.
```suggestion
const shouldApplyFalse: any = {
  id: 'custom.shouldApplyFalse', // Match field properties
  path: 'shouldApplyFalse', // Match field properties
  isCustom: true,
  process: (value: any) => value,
  shouldApply: () => false,
};
```